jazzmin/templates/admin/change_form_object_tools.html

```diff
@@ -2,9 +2,7 @@
 {% block object-tools-items %}
     <div>
         {% url opts|admin_urlname:'history' original.pk|admin_urlquote as history_url %}
-        <form action="{% add_preserved_filters history_url %}">
-            <button type="submit" class="btn btn-block btn-outline-secondary btn-sm">{% trans 'History' %}</button>
-        </form>
+        <button class="btn btn-block btn-outline-secondary btn-sm" onclick="location.href='{% add_preserved_filters history_url %}'" type="button">{% trans 'History' %}</button>
```

Comment:
I went for that option first, but it was modifying the color of the text in the button. If I go that way, I would have to add also an extra style only for that.. 

I think it would make sense to add it in case we end up having this kind of link/buttons in several places, but for a single one, this looks less messy. What do you think?